index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title>Portfolio: Setup and Mobile First</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <div class="mylogo">
+                <a href="#">My Logo</a>
+            </div>
+            <div class="union">
+                <img src="./images/Union.png" alt="Hamburguer Menu">
+            </div>
+        </nav>
+    </header>        
+    <main>
+        <section class="headline">
+            <div class="primarytext">
+                <h1>I'm Federico Glad to see you!</h1>
+            </div>
+            <div class="bodytext">
+                <p>I'm a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+            </div>
+        </section>    
+        <section class="socialmedia">
+            <div id="supportingtext">
+                <h5>LET'S CONNECT</h5>            
+            </div>
+            <div>
+                <ul class="socialmediaicons">
+                    <li><a href="#"><img src="./images/Vector.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="./images/Linkedinicon.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="./images/Vector1.svg" alt="Mail"></a></li>
+                    <li><a href="#"><img src="./images/Vector2.svg" alt="GitHub"></a></li>
+                    <li><a href="#"><img src="./images/Vector3.svg" alt="Peace"></a></li>
+                </ul>
+            </div>
+        </section>
+    </main>
```

Comment:
Kindly, what the previous reviewer tells you is that your site must be adaptable to any type of device, so using statistic widths and positions avoid that, I recommend you use your MAIN tag as a container that can be adapted to 100% of the width with a max-width of 375px, and all the texts and images within it are based on a percentage, I hope that was clear, you can force it!

Reply:
Thank you @mgmediaweb for your feedback. I make sure to implement it right now.